### PR TITLE
Cleanup repo

### PR DESCRIPTION
**Pull request by ale8k**
We had some old docs that aren't too useful or were outdated, like asyncapi, facade and howto vault. Additionally I've renamed compose-common, so it's in the conventional naming format for multiple files. We had an old docker file that wasn't used, that's been cleaned up too and finally there was a set-version script that wasn't used. I removed this directory too.

## Description

The what and why - include a summary of the change, describe what it does, and include relevant motivation and context.

Fixes _JIRA/GitHub issue number_

## Engineering checklist
*Check only items that apply*

- [ ] Documentation updated
- [ ] Covered by unit tests
- [ ] Covered by integration tests

### CODE REVIEW

File: doc/jimm-facade.md
Comment (kian99) [inline doc/jimm-facade.md:1]: Should we rather update this doc to document the JIMM facades? Maybe just one-liners like
JIMM offers these custom facades:
- AddRelation
- ...

**Review by kian99 — APPROVED** [review]

File: doc/jimm-facade.md
Comment (ale8k) [reply doc/jimm-facade.md:1]: Honestly I don't see a point... Proper doccos in jaas-docs is better I think  idk... we can revisit doccing jaas specific later maybe?

Comment (ale8k) [reply doc/jimm-facade.md:1]: Actually lets see what guys think @alesstimec @SimoneDutto

Comment (SimoneDutto) [reply doc/jimm-facade.md:1]: since we have the godoc in JIMM methods i dont' see the point of having this doc

**Review by SimoneDutto — APPROVED** [review]

File: doc/jimm-facade.md
Comment (kian99) [reply doc/jimm-facade.md:1]: Sounds good, can cut it.

**Comment by ale8k** [comment]
OKi, will merge @SimoneDutto @kian99

**Pull request merged**